src/index.tsx

```diff
@@ -743,21 +744,25 @@ const ModalizeBase = (
       scrollEnabled,
       keyboardDismissMode,
     };
-
+    const optsWithOutPanGesture = {
```

Comment:
I believe this is unnecessary since the `enabled` props on the `PanGestureHandler` is higher in the priority order. Only using `enabled` should be enough.